plugin/evm/gossiper.go

```diff
@@ -259,7 +270,24 @@ func (n *pushGossiper) awaitEthTxGossip() {
 						"len(txs)", attempted,
 						"err", err,
 					)
+					continue
+				}
+
+				gossipTxs := make([]*GossipEthTx, 0, len(txs))
+				for _, tx := range txs {
+					gossipTxs = append(gossipTxs, &GossipEthTx{Tx: tx})
+				}
+
+				n.ethTxGossiper.Add(gossipTxs...)
+				if err := n.ethTxGossiper.Gossip(context.TODO()); err != nil {
+					log.Warn(
+						"failed to send eth transactions",
+						"len(txs)", len(txs),
+						"err", err,
+					)
+					continue
```

Comment:
This `continue` seems useless right?
```suggestion
```

Reply:
It is a no-op, it's just there because I feel like we might add code after this line in the future and unintentionally run it if this errors.

Reply:
It's a no-op.... I left it in because i was worried that there might be a world where we add more code and an error would result in stuff running that we didn't intend to run. I am fine removing this though